ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -0,0 +1,70 @@
+import json
+import pathlib
+from mlagents.trainers.buffer import Buffer
+from mlagents.envs.brain import BrainParameters, BrainInfo
+
+
+class DemonstrationLoader(object):
+    @staticmethod
+    def load(file_path, brain_name, sequence_length):
```

Comment:
Is there a reason `load` has to be a method of a class? Python (unlike C#) allows functions to exist outside of classes. Seems to me that the indirection is unnecessary.